diff --git a/node_modules/@walletconnect/web3-provider/dist/cjs/index.js b/node_modules/@walletconnect/web3-provider/dist/cjs/index.js
index d177253..4d415bd 100644
--- a/node_modules/@walletconnect/web3-provider/dist/cjs/index.js
+++ b/node_modules/@walletconnect/web3-provider/dist/cjs/index.js
@@ -336,8 +336,8 @@ class WalletConnectProvider extends ProviderEngine {
         this.addProvider({
             handleRequest: (payload, next, end) => tslib_1.__awaiter(this, void 0, void 0, function* () {
                 try {
-                    const { result } = yield this.handleRequest(payload);
-                    end(null, result);
+                    const { error, result } = yield this.handleRequest(payload);
+                    end(error, result);
                 }
                 catch (error) {
                     end(error);
